src/lexer.js

```diff
@@ -292,6 +292,24 @@ export class Scanner {
   }
 }
 
+export function addIdentifierCharacters(chars) {
+  const codes = new Set();
+  for (var i = 0; i < chars.length; i++) {
+    codes.add(chars.codePointAt(i));
+  }
+
+  const _isIdentifierStart = isIdentifierStart;
+  const _isIdentifierPart = isIdentifierPart;
+
+  isIdentifierStart = function (code) {
```

Comment:
`function isIdentiferStart...` should change to `let isIdentiferStart = ...` to signal that it's value could change. Same for `isIdentifierPart`.

Reply:
Fixed